profile/ro-crate-metadata.json

```diff
@@ -1,30 +1,54 @@
 {
-  "@context": "https://w3id.org/ro/crate/1.2-DRAFT/context",
+  "@context": "https://w3id.org/ro/crate/1.2/context",
```

Comment:
Think parts of https://github.com/nfdi4plants/isa-ro-crate-profile/pull/64#discussion_r2528052204 still need addressing, particularly the second one for this crate:
* termCode SHOULD be valid as a key in JSON-LD @context of conforming RO-Crates - meaning the ISA profile either needs to define a context that conforming crates can import (see how this is done in [Workflow Run Crate](https://www.researchobject.org/workflow-run-crate/profiles/workflow_run_crate/)), OR tell users to include these terms in the context themselves. (I think the first option is preferable.)
* The term SHOULD be mapped to the term's @id in the @context of this Profile Crate